plugins/modules/net_tools/nmcli.py

```diff
@@ -2100,8 +2100,12 @@ def _compare_conn_params(self, conn_info, options):
                 if key == self.mtu_setting and self.mtu is None:
                     self.mtu = 0
                 if key == 'vpn.data':
-                    current_value = list(map(str.strip, current_value.split(',')))
-                    value = list(map(str.strip, value.split(',')))
+                    current_values = []
+                    for item in current_value.split(','):
+                        name, value_ = map(str.strip, item.split('=', 1))
+                        current_values.append('%s=%s' % (name, value_))
```

Comment:
How about this instead:
```suggestion
                        current_values.append('='.join([part.strip() for part in item.split('=', 1)]))
```
This works both for `=` being present and not.

Reply:
Thanks! I figured out an even shorter way in the latest commit.